src/sql/plot.py

```diff
@@ -612,10 +586,6 @@ def _histogram_stacked(
         cases=cases,
     )
 
-    if with_:
-        query = str(store.render(query, with_=with_))
-
-    query = conn._transpile_query(query)
-    data = conn.session.execute(sqlalchemy.sql.text(query)).fetchall()
```

Comment:
Probably I am missing something, my question is why we remove the transpile_query part?

Reply:
The logic looks good to me.
We also make sure we don't transpile the user's query.